go/vt/vtctl/workflow/server.go

```diff
@@ -971,7 +977,19 @@ func (s *Server) getWorkflowState(ctx context.Context, targetKeyspace, workflowN
 		}
 		table := ts.Tables()[0]
 
-		if ts.isPartialMigration { // shard level traffic switching is all or nothing
+		if ts.IsMultiTenantMigration() {
```

Comment:
It's worth putting an "all or nothing" comment on this line similar to what we have for shard level traffic switching.